libminifi/test/integration/OnScheduleErrorHandlingTests.cpp

```diff
@@ -29,32 +27,22 @@ class OnScheduleErrorHandlingTests : public IntegrationBase {
  public:
   virtual void runAssertions() {
     std::string logs = LogTestController::getInstance().log_output.str();
-    size_t pos = 0;
-    size_t last_pos = 0;
-    unsigned int occurances = 0;
-    do {
-      pos = logs.find(minifi::processors::KamikazeProcessor::OnScheduleExceptionStr, pos);
-      if (pos != std::string::npos) {
-        last_pos = pos;
-        pos = logs.find(minifi::processors::KamikazeProcessor::OnUnScheduleLogStr, pos);
-        if (pos != std::string::npos) {
-          last_pos = pos;
-          occurances++;
-        }
-      }
-    } while (pos != std::string::npos);
 
-    assert(occurances > 1);  // Verify retry of onSchedule and onUnSchedule calls
+    auto result = countPatInStr(logs, minifi::processors::KamikazeProcessor::OnScheduleExceptionStr);
+    size_t last_pos = result.first;
+    unsigned int occurances = result.second;
```

Comment:
Will need to change this to uint32_t as well

Reply:
typo: it's spelled occurence